From 7421a61ae9a420d82cebb02f58b5762442c44d60 Mon Sep 17 00:00:00 2001
From: Nick Kralevich <nnk@google.com>
Date: Fri, 30 Jan 2015 17:38:06 -0800
Subject: [PATCH 06/20] ensure /data/tombstones exists on all Android device.
 DO NOT MERGE

/data/tombstones is referenced by core platform code, but is not
guaranteed to exist on all Android devices. Move the directory
creation out of device specific files and into the core
init.rc file.

Bug: https://code.google.com/p/android/issues/detail?id=93207
Change-Id: I94ae5199a6a32c4fe555ca994fc4a8345e0c9690

Bug: http://b/26403620
(cherry picked from commit 0359d778a564a5c63baba08a33484fb69eda4d1e)
(cherry picked from commit 9c0ed663138138bb40396bfd350f137e5057277b)

Change-Id: If92296ed15f595b8b35ed71a9be4e96973442e0f
---
 rootdir/init.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 1ee9fbf..0c67795 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -232,6 +232,7 @@ on post-fs-data
     mkdir /data/property 0700 root root
     mkdir /data/ssh 0750 root shell
     mkdir /data/ssh/empty 0700 root root
+    mkdir /data/tombstones 0771 system system
 
     # create dalvik-cache, so as to enforce our permissions
     mkdir /data/dalvik-cache 0771 system system
-- 
2.7.4

